server.js

```diff
@@ -134,9 +134,9 @@ app.get('/api/oauth-redirect', (req, res) => { //redirects to the url configured
         })
         .then((contributorInfo) => {
             if (contributorInfo.newUser) {
-                return res.redirect(`${SITE_ROOT}/onboarding`)
+                return res.redirect(`${SITE_ROOT}/${process.env.OAUTH_REDIRECT_NEW_USERS}`)
             }
-            res.redirect(`${SITE_ROOT}/dashboard`)
+            res.redirect(`${SITE_ROOT}/${process.env.OAUTH_REDIRECT_EXISTING_USERS}`)
```

Comment:
Update `.env.example` with `OAUTH_REDIRECT_NEW_USERS` and `OAUTH_REDIRECT_EXISTING_USERS`